app/bundles/CoreBundle/Templating/Helper/TranslatorHelper.php

```diff
@@ -46,4 +46,23 @@ public function transConditional($preferred, $alternative, $parameters = array()
     {
         return $this->translator->transConditional($preferred, $alternative, $parameters, $domain, $locale);
     }
+
+    public function getJsLang()
+    {
+        $this->translator->addResource('mautic', null, $this->translator->getLocale(), 'javascript');
```

Comment:
@alanhartless Should this also add `en_US` as a fallback lang if `getLocale()` returns something other than `en_US`?